UI/window-basic-main.cpp

```diff
@@ -7631,8 +7631,10 @@ void OBSBasic::IconActivated(QSystemTrayIcon::ActivationReason reason)
 				 SLOT(OpenStudioProgramProjector()));
 
 	if (reason == QSystemTrayIcon::Trigger) {
+#ifndef __APPLE__
```

Comment:
This ifdef should probably encompass the whole if statement block itself rather than the contents of the if statement block

Reply:
True, seems like a remainder of my attempts to keep the current functionality with a modifier key - will amend this tomorrow.